Base/QTGUI/qSlicerExportNodeDialog.cxx

```diff
@@ -793,7 +841,8 @@ void qSlicerExportNodeDialogPrivate::adjustTabbingOrder()
       }
     }
   setTabOrder(this->IncludeChildrenCheckBox,this->RecursiveChildrenCheckBox);
-  setTabOrder(this->RecursiveChildrenCheckBox, this->HardenTransformCheckBox);
+  setTabOrder(this->RecursiveChildrenCheckBox, this->PreserveHierarchyCheckBox);
```

Comment:
You can set the tab order in Qt Designer. Unless there is a specific reason to do it in code, it would be better to remove these lines and set the tab order there.

Reply:
It's easier to do it in code in this case, because the tab order gets messed up as widgets get added and removed. I found it to be pretty confusing to try to set the tab order for some things in Qt Designer and for other things in code, hence the approach I took here.

Reply:
I personally set the tab order by just having the row index values ordered from starting with lowest index at the top of the ui file. I find this to be most orderly without having to add additional lines setting tab order explicitly.

Reply:
In this case rows get added and removed as the checkboxes are manipulated, so I don't think a ui-file-based approach would work.

Reply:
OK, this is a special case that may make it easier to set tab order for all widgets in code. If you describe this in a comment then it is good enough for me.